.rubocop.yml

```diff
@@ -13,6 +13,10 @@ Metrics/BlockLength:
     - 'spec/models/*'
     - 'spec/routes/*'
 
+Metrics/MethodLength:
+  Exclude:
+    - 'app/controllers/dashboard_controller.rb'
+
```

Comment:
There's no need to exclude this, we should try to keep as much as possible Rubocop's metrics for the `app` folder

Reply:
Better to update the maximum number of lines for a method.